thesheriff/domain/raid/raid.py

```diff
@@ -19,21 +19,21 @@ class Raid:
     :type gang: Gang.
     :param date: Date and time when the raid happens.
     :type date: datetime.datetime.
-    :param id: Optional, Raid Id.
-    :type id: Integer.
+    :param raid_id: Optional, Raid Id.
+    :type raid_id: Integer.
     """
 
     def __init__(
         self, name: str, location: str,
-        sheriff: Sheriff, gang: Gang, date: datetime
+        sheriff: Sheriff, gang: Gang, date: datetime, raid_id: Optional[int]
```

Comment:
Declaring it as `raid_id: Optional[int] = None` should save you from having to add None when calling the constructor without a `raid_id`.